compat/Data/ByteString/Lazy_.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE CPP #-}
+
+module Data.ByteString.Lazy_
```

Comment:
MicroHs chooses the installed module from `bytestring` instead of the local one, so I can't use `Data.ByteString.Lazy`